openghg/standardise/footprint/_paris.py

```diff
@@ -0,0 +1,175 @@
+import logging
+from pathlib import Path
+from collections import defaultdict
+from typing import DefaultDict, Dict, List, Optional, Union
+import warnings
+from xarray import Dataset
+
+from openghg.util import (
+    check_species_time_resolved,
+    check_species_lifetime,
+    timestamp_now,
+    check_function_open_nc,
+)
+from openghg.store import infer_date_range, update_zero_dim
+from openghg.types import multiPathType
+
+logger = logging.getLogger("openghg.standardise.footprint")
+logger.setLevel(logging.DEBUG)  # Have to set level for logger as well as handler
+
+
+def parse_paris(
+    filepath: multiPathType,
+    site: str,
+    domain: str,
+    model: str,
+    inlet: str,
+    species: str,
+    met_model: Optional[str] = None,
+    network: Optional[str] = None,
+    period: Optional[Union[str, tuple]] = None,
+    continuous: bool = True,
+    high_spatial_resolution: bool = False,
+    time_resolved: bool = False,
+    high_time_resolution: bool = False,
+    short_lifetime: bool = False,
+) -> Dict:
+    """
+    Read and parse input footprints data in "paris" format.
+
+    Args:
+        filepath: Path of file to load
+        site: Site name
+        domain: Domain of footprints
+        model: Model used to create footprint (e.g. NAME or FLEXPART)
+        inlet: Height above ground level in metres. Format 'NUMUNIT' e.g. "10m"
+        met_model: Underlying meteorlogical model used (e.g. UKV)
+        species: Species name. For a long-lived species this should be "inert".
+        network: Network name
+        period: Period of measurements. Only needed if this can not be inferred from the time coords
+        continuous: Whether time stamps have to be continuous.
+        high_spatial_resolution : Indicate footprints include both a low and high spatial resolution.
+        time_resolved: Indicate footprints are high time resolution (include H_back dimension)
+            Note this will be set to True automatically if species="co2" (Carbon Dioxide).
+        high_time_resolution:  This argument is deprecated and will be replaced in future versions with time_resolved.
+        short_lifetime: Indicate footprint is for a short-lived species. Needs species input.
+            Note this will be set to True if species has an associated lifetime.
+    Returns:
+        dict: Dictionary of data
+    """
+
+    if high_time_resolution:
+        warnings.warn(
+            "This argument is deprecated and will be replaced in future versions with time_resolved.",
+            DeprecationWarning,
+        )
+        time_resolved = high_time_resolution
+
+    xr_open_fn, filepath = check_function_open_nc(filepath)
+
+    with xr_open_fn(filepath) as temp:
```

Comment:
Using a closed dataset automatically re-opens it: https://github.com/pydata/xarray/issues/4131, so we don't need to use a context manager here. `fp_data = temp.load()` might let the file stay closed, I'm not sure. If you want to ensure the file is closed, you could put all of the code using `fp_data` inside the context manager.

Reply:
The main thing here is just trying to ensure we're not making this slower again so I just want to make sure we've recreated whatever this was doing with the `chunks` lines. If this wasn't closing the file anyway, that's fine and we can keep this as a simple line without the context manager.

Reply:
Ok I've updated both `paris` and `acrg_org` format to use the same method for opening the file (without the context manager).